ios/Video/Features/RCTPlayerObserver.swift

```diff
@@ -105,7 +105,9 @@ class RCTPlayerObserver: NSObject {
 
         _playerRateChangeObserver = player.observe(\.rate, options: [.old], changeHandler: _handlers.handlePlaybackRateChange)
         _playerVolumeChangeObserver = player.observe(\.volume, options: [.old], changeHandler: _handlers.handleVolumeChange)
-        _playerExternalPlaybackActiveObserver = player.observe(\.isExternalPlaybackActive, changeHandler: _handlers.handleExternalPlaybackActiveChange)
+        #if !os(visionOS)
```

Comment:
General question, Im not sure if the compiler conditionals should indent code. 

For me the default Xcode formatted (CTRL + I) did this: 

```swift
class TestClass {
  func methodTest() {
#if os(visionOS)
    print("code")
#endif
  }
}
```

(Indented the conditionals to start at the beginning)

Reply:
It's because of `swiftformat` that we are using as formatter. Generally I like is as it's more readable